api/src/services/assessmentService.ts

```diff
@@ -346,7 +346,11 @@ export const getCurriculumAssessmentBasedOnRole = async (
   principalId: number,
   assessmentId: number,
   submissionId: number
-) => {
+): Promise<{
+  curriculum_assessment: CurriculumAssessment;
+  program_assessment: ProgramAssessment;
+  assessment_submission: AssessmentSubmission;
+}> => {
```

Comment:
Please remove your changes to `assessmentService.ts` from your branch for this issue. Your issue should only have one file changed: `middleware/__tests__/assessmentsRouter.ts`.